CHANGELOG.rst

```diff
@@ -1,12 +1,15 @@
 #########
 Changelog
 #########
-All notable changes to the flow_stats NApp will be documented in this file.
+All notable changes to the kytos_stats NApp will be documented in this file.
 
 [UNRELEASED] - Under development
 ********************************
 Added
 =====
+- Event ``kytos/of_core.table_stats.received``.
+- Update `scripts/kytos_zabbix.py` script to add table stats.
```

Comment:
@gretelliz did you commit `scripts/kytos_zabbix.py` changes? It's not in the diffs.

Reply:
@viniarck, zabbix is now included. Here some examples after the update:

Option 6 is the new one [for tables]()

Number of flows:
```
# ./kytos_zabbix.py -o 5 | python3 -m json.tool
512
```
Number of flows for dpid="00:00:00:00:00:00:00:01"
```
#  ./kytos_zabbix.py -o 5 -t "00:00:00:00:00:00:00:01"| python3 -m json.tool
50
```

Number of tables:
```
# ./kytos_zabbix.py -o 6 | python3 -m json.tool
2540
```
Number of tables for dpid="00:00:00:00:00:00:00:01"
```
# ./kytos_zabbix.py -o 6 -t "00:00:00:00:00:00:00:01"| python3 -m json.tool
254
```

Reply:
@viniarck, I agree with you. I will wait for the opinion of @italovalcy to update Zabbix.

Reply:
Hi @viniarck and @gretelliz. It sounds good to have the `active_count` for now. We use that information for capacity planning. Knowing the maximum number of flows supported by a particular switch and the currently installed flows, we can plan for increasing capacity or limiting usage.

In the future, we can improve this as new requirements arrive.

Reply:
Right, Italo. @gretelliz if you could map a new task for this one that'd be great since currently `active_count` on zabbix wrapper isn't covered (being printed).

Reply:
@gretelliz, also [since PR has landed](https://github.com/kytos-ng/flow_stats/pull/56/files), feel free to close this one here. Thanks.

Reply:
@italovalcy, fine. I have a question `active_count` is a field per switch and table_id, how is it more useful? 1) report the sum of this field of all the tables of a switch, report a list with the table:field of the switch or wait for both swtich and table_id to be specified